plugins/modules/ovirt_instance_type.py

```diff
@@ -72,6 +72,7 @@
         description:
             - Operating system of the Instance Type.
```

Comment:
I'd just rephrase here:
`Operating system of the Instance Type, for example 'rhel_8x64'`